nominees/surveystack.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "SurveyStack",
+  "aliases": [
+    ""
+  ],
+  "description": "Survey Software for Community Knowledge Building",
```

Comment:
Could we get a more detailed description on what this platform is oriented around? There is are good descriptions on the Survey Stack website - would be helpful to have this here.

Reply:
Resolved in commit 4387d36